tests/Load/confirmUser.js

```diff
@@ -0,0 +1,58 @@
+import http from 'k6/http';
+import {ScenarioUtils} from "./utils/scenarioUtils.js";
+import faker from "k6/x/faker";
+import {check} from 'k6';
+import {Utils} from "./utils/utils.js";
+import {MailCatcherUtils} from "./utils/mailCatcherUtils.js";
+
+const utils = new Utils();
+const scenarioName = 'confirmUser';
+const scenarioUtils = new ScenarioUtils(utils, scenarioName);
+const mailCatcherUtils = new MailCatcherUtils(utils);
+export const options = scenarioUtils.getOptions();
+
+export default function () {
```

Comment:
looks strange, we can return confirmUser like export default async function confirmUser(), need to update everywhere